src/about/help.gemini

```diff
@@ -149,25 +167,22 @@ with fancy Unicode quotes, which include the following:
 * [Interpret (preformatted text only)] will allow Kristall to interpret ANSI escape sequences in *preformatted* text blocks.
 * [Strip] is a hybrid of the above. It strips ANSI sequences from the text, and displays the text normally, as if the escape sequences were not there.
 
-[Max. Number of Redirections] is a setting that allows you to restrict sites to redirect you only a certain number of times before erroring out. Setting this to 0 will disable redirections completely, displaying an error with the target URL.
+[Enable text highlights] allows you to enable *bolding* and _underlining_ in text/gemini documents. Bolding **like this** also works.
 
-[Redirection Handling] allows you to fine-tune the way Kristall allows redirections. Each of the options defines if Kristall should ask you to allow the redirect or do it silently. [Ask for cross-scheme redirection] will pop up a message box if a host tries to redirect you from one URL scheme to another, e.g. when a web server redirects you from HTTP to HTTPS. [Ask for cross-host redirection] will pop up the message box for all redirections through host boundaries, e.g. when example.com redirects you to www.example.com. [Ask for cross-scheme or cross-host redirection] will enable both of the previous behaviours, asking when any cross-boundary redirection happens. [Ask for all redirections] will pop up a message box every time a server tries to redirect you, keeping you in full control over all redirections. [Silently redirect everything] is the exact oppositve of that, accepting all redirections without warning or notice.
+[Gopher Map] allows you to chose a modern iconized style for gopher maps or, if you are an old schooler, just use a textual description of the item types in the map.
 
-[Network Timeout] is the time a server is allowed to *not respond anything* before a error message appears. As long as a server dripples some bytes to Kristall, no timeout will happen, so having a slow or bad connection shouldn't yield timeouts.
+[Hidden files in file:// directories] determines whether hidden files will display in local directory listings (i.e file:// URLs which do not point to a specific document but rather a directory).
 
-[Additional Toolbar Items] contains various additional toolbar items which some may find useful.
+[Strip <nav> from HTML pages] allows you to remove the <nav> tag from HTML pages. This might make certain pages more readable, others completly unusable.
 
-* [Home] button opens the configured home page in the current tab.
-* [New tab] button appears to the right of the tab bar. This simply adds a new tab to the current window.
-* [Root] button takes you to the root directory of the current site. (See Menus>Navigation section for explanation of what this does).
-* [Parent] button takes you to the parent directory of the current site. (See Menus>Navigation section for explanation of what this does).
+### Cache
```

Comment:
Note: there is also a "Caching" section near the bottom of the help document too. There's a bit of duplicated info so I think we should get rid of the old one and add a little more info here. I'll send in a PR

Reply:
Actually, I just realised that the section you added is fine for describing the options in the tab, and the two sections have fairly different content in them. Disregard what I said above 😆